app/assets/stylesheets/application.scss

```diff
@@ -10,5 +10,42 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- */
-@import 'bootstrap';
\ No newline at end of file
+*/
+
+/* https://github.com/twbs/bootstrap-rubygem */
+@import 'bootstrap';
+@import 'rails_bootstrap_forms';
+
+/* forms */
+
+$state-danger-text: #c00;
+
+input, textarea, select, .uneditable-input {
+  border: 1px solid #bbb;
+  width: 100%;
+  margin-bottom: 15px;
+  box-sizing: border-box;
+}
+
+input {
+  height: auto !important;
+}
+
+input[type=submit] {
+  width: 33%;
+}
+
+#error_explanation {
+  color: red;
+  ul {
+    color: red;
+    margin: 0 0 30px 0;
+  }
+}
+
+.field_with_errors {
+  // @extend .has-error;
```

Comment:
not sure where you got these snippets from, but it is unable to extend the `has-error` class because it does not exist

ditto $state-danger-text, which is why I redefined it above